src/models/Deployment.ts

```diff
@@ -51,7 +51,7 @@ export class Deployment implements IDeployment {
   public entrypoint: string | null
   public storageDocumentId: string | null
   public infrastructureDocumentId: string | null
-  public infrastructureOverrides: Record<string, unknown> | null
+  public infrastructureOverrides: unknown | string | null
```

Comment:
I think we probably don't want to change this type. But in the component we can convert it to json and back.